Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust
+
+RUN apt-get update && \
+    apt-get install -y cmake && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . /opt/tarpaulin/
+
+RUN cd /opt/tarpaulin/ && \
+    cargo install && \
+    rm -rf /opt/tarpaulin/ && \
+    rm -rf /usr/local/cargo/registry/
+
+WORKDIR /volume
```

Comment:
This is where the command is going to be run. The `-v "$PWD:/volume"` argument to `docker run` maps the working directory outside docker to `/volume`.